packages/wdio-browser-runner/src/index.ts

```diff
@@ -98,7 +97,8 @@ export default class BrowserRunner extends LocalRunner {
 
         try {
             await server.start()
-            runArgs.args.baseUrl = `http://localhost:${server.config.server?.port}`
+            const hostname = this.#options.hostname || DEFAULT_HOSTNAME
+            runArgs.args.baseUrl = `http://${hostname}:${server.config.server?.port}`
```

Comment:
With this change, will it only start allowing specified `hostnames`, but still not allow specifying `https`? I agree from your comment on the issue that `port` doesn't need to be customizable, but what about `https`?

Reply:
@HassHz what do you think of calling the option `host` and have the user include protocol and hostname?

Reply:
I believe a combined `host` would work fine for my usage. Would there be a default certificates?
I wonder if having separate `https` and `hostname` option would be needed for others' use cases though... maybe if for whatever reason they need to provide secure certs? It's something the Vite allows, though.
`https: true` or:
```
https: {
  key: ...,
  cert: ...
}
```

Reply:
> Would there be some default self-signed certificates?

No, we can combine it to host as well 👍